vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System.ComponentModel.Composition
+open System.Runtime.InteropServices
+
+open Microsoft.VisualStudio.FSharp.UIResources
+open SettingsPersistence
+open OptionsUIHelpers
+
+//autoproperties can be used to conveniently define defaults and faciliate data binding,
+//but the properties shouldn't be mutated outside of the Options dialog UI
+type IntelliSenseOptions() =
```

Comment:
Now I see how confusing this is. I'll add a better comment and maybe an example with immutable type.